Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM python:3.12-slim
+LABEL maintainer="denys.kotovskiy@gmail.com"
+
+ENV PYTHONUNBUFFERED 1
+
+WORKDIR /app
+
+COPY requirements.txt requirements.txt
+RUN pip install -r requirements.txt
+
+COPY . .
+RUN mkdir -p /files/media
```

Comment:
The `RUN mkdir -p /files/media` command is correctly creating the directory, but ensure that the application logic actually requires this directory to be created at this stage.